projects/canopy/src/lib/carousel/carousel-item/carousel-item.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component, HostBinding } from '@angular/core';
+@Component({
+  selector: 'lg-carousel-item',
+  templateUrl: './carousel-item.component.html',
+  styleUrls: ['./carousel-item.component.scss'],
+})
+export class LgCarouselItemComponent {
+  @HostBinding('class') classes = 'lg-carousel-item lg-padding-md';
```

Comment:
We could probably have a unit test to validate the class is applied. We've done similar things on other components if you want to refer to them